content/en/blog/2023/notary-completes-fuzzing-security-audit.md

```diff
@@ -0,0 +1,44 @@
+---
+title: The Notary Project completes fuzzing security audit
+author:  "Adam Korczynski, David Korczynski, and Feynman Zhou"
+date: 2023-03-17
+draft: false
+---
+
+> Reviewed by Pritesh Bandi, Samir Kakkar, Shiwei Zhang, Toddy Mladenov, Vani Rao, Yi Zha
+
+The Notary Project is happy to announce the completion of its fuzzing security audit. The audit was carried out by Ada Logics and is part of [an initiative by the CNCF](https://www.cncf.io/blog/2022/06/28/improving-security-by-fuzzing-the-cncf-landscape/) to bring fuzzing to the CNCF landscape. The audit spanned several months in late 2022 and early 2023 and resulted in 20 fuzzers written for 3 Notary sub-projects and 2 issues found, one of which was security-critical. 
```

Comment:
Consider:
The audit spanned several months in late 2022 and early 2023 and resulted in 20 fuzzers written for 3 Notary sub-projects and 2 issues being identified and addressed including a critical security fix.

Reply:
Updated